Meadowlark-js/packages/meadowlark-core/src/validation/ResourceValidator.ts

```diff
@@ -38,6 +38,20 @@ export type ResourceValidationResult = {
 export async function validateResource(pathComponents: PathComponents): Promise<ResourceValidationResult> {
   // Equally supporting resources with either upper or lower case names
   const lowerResourceName = decapitalize(pathComponents.resourceName);
+
+  // Manually exclude schoolYearTypes as a valid endpoint
+  if (lowerResourceName === 'schoolYearTypes') {
+    const invalidResourceMessage = 'This Ed-Fi API instance does not support managing schoolYearType as a resource.';
+    return {
+      headerMetadata: {},
+      resourceName: pathComponents.resourceName,
+      resourceInfo: NoResourceInfo,
+      errorBody: JSON.stringify({
+        message: invalidResourceMessage,
+      }),
+    };
+  }
+
```

Comment:
There's a better place for this. In ResourceNameMapping, school year enumeration is added to the resource lookup mapping. Deleting that code block should do you you want.